fe/src/main/java/org/apache/doris/qe/Coordinator.java

```diff
@@ -882,12 +889,60 @@ private void computeFragmentHosts() throws Exception {
         }
     }
 
-    //One fragment could only have one HashJoinNode
-    private boolean isColocateJoin(PlanNode node) {
-        if (Config.disable_colocate_join) {
+    private boolean needParallelInstance(PlanNode leftMostNode) {
+        if (getParallelExecInstanceNum() <= SessionVariable.MIN_EXEC_INSTANCE_NUM) {
```

Comment:
I prefer 1 to `SessionVariable.MIN_EXEC_INSTANCE_NUM`

Reply:
OK